package/MDAnalysis/coordinates/MOL2.py

```diff
@@ -373,16 +373,7 @@ def encode_block(self, obj):
         molecule[1] = molecule_1_store
         return return_val
 
-    def write(self, obj):
-        """Write a new frame to the MOL2 file.
-
-        Parameters
-        ----------
-        obj : AtomGroup or Universe
-        """
-        self.write_next_timestep(obj)
-
-    def write_next_timestep(self, obj):
+    def _write_next_frame(self, obj):
         """Write a new frame to the MOL2 file.
 
         Parameters
```

Comment:
Same. Converting `obj = obj.atoms` should do it.

Reply:
I've raised #2715 for this.  In general the MOL2 writer isn't terrific.

Reply:
Here it's because `atomgroup_intersection` wants a `.indices` attribute which is present in an AtomGroup but not a Universe, instead of inherent mol2 problems. If you safeguard it by converting `obj` to `obj.atoms` in the method that should fix it.

Reply:
I opened a separate PR/Issue set for the universe/ag reading issue #2718
If we can get it in for 1.0 too that would be great.